pkgpanda/build/__init__.py

```diff
@@ -19,8 +19,8 @@
 from pkgpanda.constants import RESERVED_UNIT_NAMES
 from pkgpanda.exceptions import FetchError, PackageError, ValidationError
 from pkgpanda.util import (check_forbidden_services, download_atomic, load_json,
-                           load_string, make_file, make_tar, rewrite_symlinks,
-                           write_json, write_string)
+                           load_string, logger, make_file, make_tar,
```

Comment:
minor request: Hanging indent with sorted names will be better here instead of 3 column labels in the import.